v5.31.4 docs (#4519)
mattermost/docs: source/administration/changelog.md

## Release v5.31 - [ESR](https://docs.mattermost.com/administration/extended-support-release.html)

- **v5.31.4, released 2021-04-23**

>   - Fixed an issue with short-circuit slash command hooks when a plugin consumed one for client-side processing. [MM-35074](https://mattermost.atlassian.net/browse/MM-35074)

Reviewer: ```suggestion
  - Fixed an issue with client-side slash commands being processed by multiple plugins. [MM-35074](https://mattermost.atlassian.net/browse/MM-35074)
```
